src/plugins/elasticsearch/index.js

```diff
@@ -23,7 +23,8 @@ module.exports = function (kibana) {
           cert: Joi.string(),
           key: Joi.string()
         }).default(),
-        minimumVerison: Joi.string().default('1.4.4')
+        apiVersion: Joi.string().default('master'),
```

Comment:
I talked to Spencer about this - IIRC, he couldn't get a 2.0 version of the client until there was a 2.0 branch of ES. Now that there is, he just needs to release a new version of the client that has 2.0 API support. I don't know if that's happened yet, however.